### ISSUE TITLE
logD calculations

### ISSUE THREAD

**Issue by luxsika** [OpenSourceMycetoma/Series-1-Fenarimols#14]
Hi all. I am Luxsika, a MSc student at UCL. I am currently working on the Mycetoma Project with Matthew Todd. So far, I have synthesized the compound EPL-BS0495, based on Hung's thesis. The scheme is attached below. ELN is currently not up for me, so I am waiting on UCL to sort that out for me.

![image](https://user-images.githubusercontent.com/50544220/59281386-15fa0780-8c5f-11e9-9a2a-f1c786559972.png)

My aim now is to change the chemistry of the precursor.  @bendndi attached an image of the OSN target list .
![image](https://user-images.githubusercontent.com/50544220/59281758-b6502c00-8c5f-11e9-975f-f2a8b7e947d4.png)

I have chosen 3 compounds from this image to make.  P4_A_002, P4_A_005 and P4_B_010. I have calculated the logD for these 3 compounds using Chemaxon. However, I am not too sure if they are correct. I aiming for a pH between 2.5-6. As it is difficult  with getting the compound into the grain in vivo, a range of pH will be ideal. From the Paper, attached below, it suggests that pH should be above 2.5 and I am aiming for that range so that I can validate the findings in that paper.

[calculated log D-converted.pdf](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/3277088/calculated.log.D-converted.pdf)

https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0006437

Could anyone please check the predicted logD values of these compounds?

Thank you
Luxsika

**Comment by bendndi** [comment]
@luxsika 
Thanks for the update!
For the logD / pH, calculation from Chemaxon, what is important is the logD _at pH _7.4__. BAsed on the pdf you shared it looks lik eyou ghet a good range of logD@pH7.4 for these compounds (around 2.7, 2.5, 5.0).
It's always important to keep in mind that these are calculations of logD rather than measurements. There are many different ways of calculating logD coming from different software providers and based on different underlying algoroithms, all of which would give different absolute values for the same molecules, sometiimes with quite different results. In my experience there is no particular algorithm which is globally better than another when comparing the calculated values to the experimental values. What is importnat is not so much which algorothm you use, but to make sure that you always use the same calculation method when comparing values. Would be interested to get @drc007 view on this too.

You'll note from the paper you referenced that we used Stardrop logD calculation for the plot in the paper:
![image](https://user-images.githubusercontent.com/36627399/59328601-0a552200-8ced-11e9-81e8-ad049c63c4fd.png)

I've calculated the Stardrop logD for the three compounds you selected, and they are below (you'll see the rank order is about the same but the absolute values are not the same as the chemaxon - that's because the algorithm is different):
![image](https://user-images.githubusercontent.com/36627399/59328950-fb22a400-8ced-11e9-8db5-45d737c82822.png)

I'll update the OSN master sheet elsewhere on GitHub and from now on I'll include the Stardrop logD values on the image!

**Comment by drc007** [comment]
I agree once you decide on an algorithm for calculating LogD you need to stick with it. Trying to combine results from different algorithms is very unwise.

**Comment by MFernflower** [comment]
Unrelated!!! but I feel this is important information to know before going into the lab regarding availability of drug fragments @luxsika 

https://www.sigmaaldrich.com/catalog/product/aldrich/415308?lang=en&region=US monoethylpiperazine is rather cheap so I think this would be better than the ELN route of Cl-intermediate --> couple to monoBOCpiperazine ---> deprotect ---> couple to ethyl bromide

The furnished piperazinyl urea fragment is a bit pricey and only available from oakwood chemicals so this isn't gonna help 

1-(2-Methoxyphenyl)piperazine is rather cheap from SA (sub 40 usd for amounts needed) and so is 1-(2-Fluorophenyl)piperazine

**Comment by MFernflower** [comment]
I don't have access to stardrop so thank you for taking the time!

**Comment by MFernflower** [comment]
@luxsika @dmitrij176 Links to ELN'S?

**Comment by dmitrij176** [comment]
@MFernflower ELN's are currently unavailable, but will be set up shortly

**Comment by MFernflower** [comment]
Just curious - Has any synthesis work been done? Like has the alcohol been converted to the bromide? @dmitrij176

**Comment by dmitrij176** [comment]
@MFernflower I did the following multistep synthesis, with final product variation (ethyl/beznzyl chloroformate). Apart from that, Ive been also working on 6 novel compounds (project update is in issue 13).
![59281386-15fa0780-8c5f-11e9-9a2a-f1c786559972](https://user-images.githubusercontent.com/44785501/61552832-8ae00e80-aa61-11e9-8385-dd3703f82c34.png)
